src/main/java/gmbs/domain/WinningResult.java

```diff
@@ -0,0 +1,29 @@
+package gmbs.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+public class WinningResult {
+    private final Map<Ranking, Integer> winningResult = new EnumMap<>(Ranking.class);
+
+    public WinningResult(List<Ranking> rankings) {
+        putValues(rankings);
+    }
+
+    public long calculatePrizeSum() {
+        return winningResult.entrySet().stream()
+                .mapToLong(entry -> entry.getKey().multiple(entry.getValue()))
+                .sum();
+    }
+
+    public Map<Ranking, Integer> getWinningResult() {
+        return winningResult;
+    }
+
+    private void putValues(List<Ranking> rankings) {
+        for (Ranking ranking : rankings) {
+            winningResult.put(ranking, winningResult.getOrDefault(ranking, 0) + 1);
```

Comment:
```suggestion
            winningResult.put(ranking, winningResult.get(ranking) + 1);
```
위처럼 작성할 수도 있을 것 같은데요. 어떤게 더 가독성 측면에서 좋아보이시나요~?